### DISCUSSION TITLE
ModuleNotFoundError: No module named 'MySQLdb'

### DISCUSSION THREAD

**Discussion by yaya159456** [optuna/optuna#5477] [Q&A]
### Don't use GitHub Issues to ask support questions.

when i do the command:
`optuna create-study --study-name "node_price_500kv_20240101_20240510_3d_0w_0m__astcgn" --storage "mysql://optuna:optuna123@192.168.5.13:3306/optuna"`
The ERROR has happen:
![image](https://github.com/optuna/optuna/assets/44463912/b1ae490c-5040-4c87-9e18-cff2ad161dbf)
why?

**Comment by nzw0301** [comment]
https://stackoverflow.com/questions/22252397/importerror-no-module-named-mysqldb
